pkg/deployment/resources/pod_creator.go

```diff
@@ -217,9 +218,77 @@ func createArangodArgs(apiObject metav1.Object, deplSpec api.DeploymentSpec, gro
 }
 
 // createArangoSyncArgs creates command line arguments for an arangosync server in the given group.
-func createArangoSyncArgs(spec api.DeploymentSpec, group api.ServerGroup, groupSpec api.ServerGroupSpec, agents api.MemberStatusList, id string) []string {
-	// TODO
-	return nil
+func createArangoSyncArgs(apiObject metav1.Object, spec api.DeploymentSpec, group api.ServerGroup, groupSpec api.ServerGroupSpec, agents api.MemberStatusList, id string) []string {
+	options := make([]optionPair, 0, 64)
+	var runCmd string
+	var port int
+
+	/*if config.DebugCluster {
```

Comment:
Is this commented out intentionally?

Reply:
left over from starter. Leave it for now